module/zfs/abd.c

```diff
@@ -148,7 +187,15 @@ boolean_t zfs_abd_scatter_enabled = B_TRUE;
  * will cause the machine to panic if you change it and try to access the data
  * within a scattered ABD.
  */
-size_t zfs_abd_chunk_size = 4096;
+#ifdef __APPLE__
+#ifdef _KERNEL
+size_t zfs_abd_chunk_size = 1024; // original from openzfs uses 1024
+#else
+size_t zfs_abd_chunk_size = 1024;
+#endif
+#else
+size_t zfs_abd_chunk_size = 1024;
+#endif
 
```

Comment:
ah, tried 4k and 1k and different in userland vs kernel, so that can go away to just 1k now.